### DISCUSSION TITLE
transaction history

### DISCUSSION THREAD

**Discussion by Ak1ra1** [jumperexchange/jumper-exchange#783] [Ideas]
I would like to discuss the possibility of adding transaction history, which would make it easier to find the right one in the future and track routes in general 
![image](https://github.com/lifinance/jumper.exchange/assets/108584707/86fe6bea-ad57-4f39-b40b-77ff464817ee)
![image](https://github.com/lifinance/jumper.exchange/assets/108584707/a7618ca7-b27d-44be-b4f8-e4f05b3fd11f)

**Comment by chybisov** [comment]
Hey, we have it here 🙂

<img width="431" alt="image" src="https://github.com/lifinance/jumper.exchange/assets/18644653/860c1b12-e64e-4842-8fd0-509445057f27">
<img width="419" alt="image" src="https://github.com/lifinance/jumper.exchange/assets/18644653/192437d3-dfeb-487e-af9c-56949c7c1694">

**Comment by Ak1ra1** [comment]
sorry, didn't notice

![image](https://github.com/lifinance/jumper.exchange/assets/108584707/5c7cd9b4-31ff-481e-9b74-695c7817e76b)
